src/renderers/WebGLRenderer.js

```diff
@@ -987,22 +996,40 @@ function WebGLRenderer( parameters = {} ) {
 
 		if ( camera.isArrayCamera ) {
 
-			const cameras = camera.cameras;
 
-			for ( let i = 0, l = cameras.length; i < l; i ++ ) {
+			if ( xr.enabled && multiview.isAvailable() ) {
```

Comment:
Here you can just use `const l = xr.enabled && multiview.IsAvailable() ? 1 : cameras.length`.